Python/instrumentation.c

```diff
@@ -891,18 +891,42 @@ static inline int most_significant_bit(uint8_t bits) {
 static uint32_t
 global_version(PyInterpreterState *interp)
 {
-    return interp->ceval.eval_breaker & ~_PY_EVAL_EVENTS_MASK;
+    return _Py_atomic_load_uintptr_relaxed(&interp->ceval.instrumentation_version);
```

Comment:
This needs a cast to `uint32_t` now

Reply:
I can add the cast, but I don't think this changed the type of the expression being returned - `eval_breaker` was already a `uintptr_t`, so `interp->ceval.eval_breaker & ~_PY_EVAL_EVENTS_MASK` should also be `uintptr_t`.